azure/datalake/store/utils.py

```diff
@@ -86,7 +86,6 @@ def read_block(f, offset, length, delimiter=None):
         f.seek(start + length)
         seek_delimiter(f, delimiter, 2**16)
         end = f.tell()
-        eof = not f.read(1)
```

Comment:
LGTM to me, seems not used. No side effect of read? (like moving any pointer). Didn't look at the entire code to check.